geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/iterators/AttributeIndexFilteringIteratorTest.scala

```diff
@@ -65,7 +64,7 @@ class AttributeIndexFilteringIteratorTest extends Specification with TestWithDat
       // % should return all features
       val wildCardQuery = new Query(sftName, ff.like(ff.property("name"),"%"))
       checkStrategies(wildCardQuery, AttributeIndex)
-      SelfClosingIterator(fs.getFeatures()) must haveLength(16)
+      SelfClosingIterator(fs.getFeatures(wildCardQuery)) must haveLength(16)
```

Comment:
i think this was a typo in the original test case - the wildcard filter should return all features, but it wasn't actually tested. `fs.getFeatures()` won't even hit the attribute index...